contracts/Pod.sol

```diff
@@ -141,6 +147,42 @@ contract Pod is Initializable, ReentrancyGuardUpgradeSafe, ERC777UpgradeSafe, Ba
         return _assetsRedeemed;
     }
 
+    function sponsor(uint256 _amount) external nonReentrant {
+        address _sender = _msgSender();
+        Ticket _ticket = ticket();
+        IERC20 _assetToken = yieldService().token();
+
+        // Collect collateral from caller
+        _assetToken.transferFrom(_sender, address(this), _amount);
+
+        // Transfer collateral to Prize Pool for Tickets without Pod-Shares
+        _assetToken.approve(address(_ticket), _amount);
+        _ticket.mintTickets(_amount);
+
+        // Mint Sponsorship Tokens equal to Amount of Collateral supplied
+        PodSponsorship(podSponsorToken).mint(_sender, _amount);
+
+        // Log event
+        emit PodSponsored(_sender, _amount);
+    }
+
+    function redeemSponsorship(uint256 _amount) external nonReentrant {
+        address _sender = _msgSender();
+        uint256 _balance = PodSponsorship(podSponsorToken).balanceOf(_sender);
+        require(_balance >= _amount, "Pod: Insufficient sponsorship balance");
+
+        // Redeem Sponsorship Tokens for Assets
+        uint256 _assets = ticket().redeemTicketsInstantly(_amount);
```

Comment:
Ooooh this is interesting!  

Redeem tickets instantly will take a fee.  Sponsors may not want to pay a fee.

We'll have to reflect the same pattern as what is in the pool:

`redeemSponsorshipInstantly`

and

`redeemSponsorshipWithTimelock`

If they choose the instant version, we won't receive the full amount.  They'll receive the amount less the fee.